assignments/closure.js

```diff
@@ -1,11 +1,32 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+function whatGoals(){
+
+const goals = "Plenty,";
+function returnGoals (){
+  const moreGoals = "of them!";
+  console.log(goals + " I got plenty " + moreGoals);
+}
+
+returnGoals();
+
+}
+
+whatGoals();
 
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
+  let count = 0;
+  return () => (++count);
+  
+  
   // Return a function that when invoked increments and returns a counter variable.
 };
+
```

Comment:
You aren't quite finished implementing closure yet. Refer to the provided solution code to identify your missing steps.